community/io/src/main/java/org/neo4j/io/pagecache/impl/muninn/MuninnPagedFile.java

```diff
@@ -609,6 +610,25 @@ MuninnPage evictPage( long filePageId )
         return (MuninnPage) element;
     }
 
+    void setHighestEvictedTransactionId( long modifiedTransactionId )
```

Comment:
Looks like a very similar method as `MuninnPage#setLastModifiedTxId`, could be made a utility with nice javadoc?